jwst/jump/twopoint_difference.py

```diff
@@ -30,8 +31,11 @@ def find_crs(data, group_dq, read_noise, rej_threshold, nframes):
     # Get data characteristics
     (nints, ngroups, nrows, ncols) = data.shape
 
-    # Create array for output median slope images
+    # Create arrays for output
     median_slopes = np.zeros((nints, nrows, ncols), dtype=np.float32)
```

Comment:
As mentioned earlier, I don't think we really need median_slopes anymore.

Reply:
This array is used within twopoint_difference. It is not longer returned or mentioned in other routines or tests.

Reply:
needed internally but no longer is returned